src/text-editor.coffee

```diff
@@ -1310,14 +1351,27 @@ class TextEditor extends Model
   #   * `position` (optional) Only applicable to decorations of type `overlay`,
   #     controls where the overlay view is positioned relative to the marker.
   #     Values can be `'head'` (the default), or `'tail'`.
+  #   * `name` (optional) Only applicable to decorations of type `gutter`. Specifies
+  #      which gutter this applies to. If missing, the default gutter is assumed.
   #
   # Returns a {Decoration} object
   decorateMarker: (marker, decorationParams) ->
     if decorationParams.type is 'gutter'
+      # #TODO REVIEW: should we keep 'gutter', now that there can be many gutters?
+      #   For now, I'll go with 'line-number' = 'gutter'.
+      # Question 2: Looking ahead, should the API allow adding an "item" to
+      #   the default (line number) gutter? I think no makes sense, but want to run this by everyone.
```

Comment:
I don't think we should allow the `line-numbers` gutter to get decorated with items for now. Just classes.